packages/1155-contracts/src/nft/ZoraCreator1155Impl.sol

```diff
@@ -454,9 +454,20 @@ contract ZoraCreator1155Impl is
     }
 
     /// @notice Get the creator reward recipient address
-    /// @dev The creator is not enforced to set a funds recipient address, so in that case the reward would be claimable by creator's contract
-    function getCreatorRewardRecipient() public view returns (address payable) {
-        return config.fundsRecipient != address(0) ? config.fundsRecipient : payable(address(this));
+    /// @param tokenId The token id to get the creator reward recipient for
+    /// @dev The creator is not enforced to set a funds recipient address for a specific token or contract-wide,
+    ///      so in the case of both the reward recipient is set to the creator's contract,
+    ///      which can be withdrawn by the creator via the `withdrawRewards` function.
+    function getCreatorRewardRecipient(uint256 tokenId) public view returns (address payable) {
```

Comment:
@kulkarohan can we check the front-end and subgraph to make sure there are no calls to this function?

Would it make sense to make this internal?

Reply:
confirmed - no frontend and subgraph usage of this function (yet)

i believe however frontend is planning to pin the creator reward recipient address on each token page, so i think having it public to fetch would make their lives easier, at little cost to us since we need it anyways.

but can confirm w them and if they're not can just make this internal